ghost/core/test/e2e-api/members/send-magic-link.test.js

```diff
@@ -376,6 +376,19 @@ describe('sendMagicLink', function () {
                 .expectEmptyBody()
                 .expectStatus(201);
         });
+
+        it('blocks changing email to a blocked domain', async function () {
+            settingsCache.set('all_blocked_email_domains', {value: ['blocked-domain-setting.com']});
+            const email = 'hello@cool-domain-setting.com';
+            await membersService.api.members.create({email, name: 'Member Test'});
+
+            await membersAgent.post('/api/member/email/')
+                .body({
+                    email: 'hello@blocked-domain-setting.com',
+                    identity: '12345678'
```

Comment:
There's currently update member update email tests, since our E2E test suite doesn't have a way of populating / mocking the identity field.

Reply:
Nice! I think the opposite test would also be useful: 'Allows changing email to a non-blocked email domain'

Reply:
Added in a588ec6b8859ea48ab9fda2f90dc5e676364a8b7